### PR TITLE
Upgrade and clean cache

### PR DESCRIPTION
**Pull request by tramuntanal**
This PR upgrades from our latest Decidim 0.27 clean-app in order to get the clean cache scheduled task.

This task will be scheduled in the server when deploying as whenever resets all the cronjobs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,11 +21,10 @@ public/uploads/
...
 /config/puma.rb
-
-/public/sw.js.br
-/public/sw.js.gz
-/public/sw.js.map.br
-/public/sw.js.map.gz
+# Ignore deploy secrets
+/config/deploy.rb
+/config/deploy/*
+/Capfile
```
Comment (laurajaime) [inline .gitignore:27]: this files not exists here

File: .rubocop_ruby.yml [hunk 2]
```diff
@@ -70,10 +70,6 @@ AllCops:
   # Otherwise we fallback to the oldest officially supported Ruby version (2.0).
   TargetRubyVersion: 2.7
 
-  RSpec:
-    Patterns:
-      - "(?:^|/)spec/"
-      - "(?:^|/)test/"
```
Comment (laurajaime) [inline .rubocop_ruby.yml:76]: do you want to remove it?

File: Gemfile [hunk 3]
```diff
@@ -43,8 +44,7 @@ gem "whenever"
 # endif
 
 group :development, :test do
-  # Fixed to 2.9.1 version in order to avoid sassc error
-  gem "better_errors", "~> 2.9.1"
+  gem "better_errors"
```
Comment (laurajaime) [inline Gemfile:47]: The sassc error no longer happen?

File: README.md
Comment (laurajaime) [inline README.md:1]: do you want references to clean app here?

File: config/initializers/decidim.rb [hunk 4]
```diff
@@ -165,9 +165,6 @@
   config.follow_http_x_forwarded_host = Rails.application.secrets.decidim[:follow_http_x_forwarded_host].present?
 end
 
-# Inform Decidim about the assets folder
-Decidim.register_assets_path File.expand_path("app/packs", Rails.application.root)
-
```
Comment (laurajaime) [inline config/initializers/decidim.rb:170]: be careful

File: docs/how_to_upgrade.md
Comment (laurajaime) [inline docs/how_to_upgrade.md:1]: This folder doc I think must be removed from here

**Review by laurajaime — COMMENTED** [review]
Please, check my comments

File: .rubocop_ruby.yml (hunk 2 above)
Comment (tramuntanal) [reply .rubocop_ruby.yml:76]: yes, it is causing rubocop errors and does not add anything

File: Gemfile (hunk 3 above)
Comment (tramuntanal) [reply Gemfile:47]: no, it works in development and test

File: README.md
Comment (tramuntanal) [reply README.md:1]: I prefer to minimize merge conflicts with clean-app

**Review by laurajaime — APPROVED** [review]

**Pull request merged**